docs/client-side/yeoman.rst

```diff
@@ -0,0 +1,260 @@
+
+9.	The generator prompts for the name of the application. Leave the default and press ``Enter``.
+
+.. image:: yeoman/_static/yeoman-yo-aspnet.png
+
+10.	Open Visual Studio 2015. From the File menu select :menuselection:`Open --> Project/Solution`.
+
+11.	In the Open Project dialog, navigate to the ``project.json`` file, select it and click the **Open** button. In the Solution Explorer, the project should look something like the screenshot below.
+
+ .. image:: yeoman/_static/yeoman-empty-application.png
+ 
+12.	This example doesn't use the cross-platform Kestrel server, so open ``project.json`` for editing and remove both references to **kestrel** from the **dependencies** and **commands** sections. Save the ``project.json`` file.
+
+13.	Press **F5** to run the project. The default Welcome screen should display.
+ 
+ .. image:: yeoman/_static/yeoman-welcome.png 
+ 
+Integrating Generated Projects with Visual Studio
+-------------------------------------------------
+
+Generator authors may not have Visual Studio in mind when they create a new Yeoman generator. Each generator implements its own distinct, opinionated, grouping of technology. As long as the generator creates a working set of files for the purpose, the generator is successful. For example, the `Yeoman team <http://yeoman.io/>`_ has a webapp generator that scaffolds a front-end web application. The generator includes a build process with Grunt or Gulp and Bower and a number of other features. The generated files have no concept of "project". The generated files (see screenshot below) don't contain a `project.json` and you won't find project files that are specific to Visual Studio.
+
+ .. image:: yeoman/_static/yeoman-generated-files.png  
+  
+To work on Yeoman generated files in Visual Studio, you have options:
+ 
+- Leave the file structure exactly as generated and use the Visual Studio :menuselection:`File --> Open --> Web Site` option to populate the solution as shown in the screenshot below. The web site option is the safest route for getting the project to work in Visual Studio initially. You can test the site from the command line before adding it to Visual Studio. If the site runs separately, it will run as a web site in Visual Studio. The Grunt and Gulp client-build tools will still work in Visual Studio, but, at the time of this writing, the **Dependencies** folder for NPM and Bower will not appear automatically. 
+ 
+ .. image:: yeoman/_static/yeoman-yowebapp-solution.png  
+  
+- If the generator creates a project.json file, you can open these from the Visual Studio :menuselection:`File --> Open --> Project/Solution...` and then select the generated `project.json` (ASP.NET 5 Project File type). 
+
+- If you want the integration and tooling supplied by a Visual Studio project but the generator doesn't produce a project.json file, you can create a Visual Studio project and copy the client build and application files into the project.
+
+Generating a Webapp
```

Comment:
My current game plan - let me know if this is what you want: 
-Reference the http://yeoman.io/learning/ for initial toolset. 
-npm install generator-aspnet
-generate Web Application type
-Show project loading and running in VS 2015
-Use command line dnx/dnu to restore, build and run through web and kestrel. 
-Show sub generator to add a new class.